doc/dependencies.csv

```diff
@@ -2,29 +2,22 @@ Build option/target,Tool name,Minimum version,Notes,Update criteria
 --,CMake,3.19,,"* Is available as a module on HPC systems
 * When arbitrary changes in more recently released versions make supported"
 --,C++ compiler,,C++17 support. See below,
---,GCC,9.0.0,,"* it is supported by the minimum version of CUDA.
+--,GCC,12.0.0,,"* it is supported by the minimum version of CUDA.
 * it is available either by default or can be installed using standard package manager on the supported Linux versions."
---,Clang,10.0,Needs GCC 9 or later for standard library.,
---,Apple Clang,11,Apple LLVM version 11.0.0 (clang-1100.0.33.8),
+--,Clang,12.0,,
+--,Apple Clang,13,Apple LLVM version 13.0.0 (clang-1300.x.y.z),
```

Comment:
how do I find the more recent Clang version supported by Apple? I read something here https://forums.developer.apple.com/forums/thread/746509

But the numbers do not match exactly

Reply:
That's a lot of manual I am afraid. I source the versions here
https://en.wikipedia.org/wiki/Xcode
https://en.wikipedia.org/wiki/Clang
https://gcc.gnu.org/releases.html

and for CUDA I use the NVIDIA blog (I found no handy table, sadly)